addons/html_editor/static/src/utils/dom.js

```diff
@@ -201,15 +201,15 @@ export function fillShrunkPhrasingParent(el) {
  * @returns {HTMLElement|undefined} the removed br, if any
  */
 export function cleanTrailingBR(el) {
-    let br;
-    const candidate = el.lastChild;
-    if (candidate && candidate.tagName === "BR" && !isEmptyBlock(el)) {
-        if (candidate.previousSibling?.tagName !== "BR") {
-            br = candidate;
-            candidate.remove();
-        }
+    const candidate = el?.lastChild;
```

Comment:
Why el could be undefined ?

Reply:
domInsert calls cleanTrailingBR with `node.parentElement`, being `node` a disconnected text node in some cases.
Ideally I would have to debug domInsert, which would probably be very time consuming. (IMO it has to be re-written)
So I just made a fix that prevents the traceback.
The previous implementation of this function before ged's optimization called `isEmptyBlock` as the first thing, which implicitily protected the function against null argument. (I didn't think it was a story worth telling in the commit message, but if you prefer, I can include it).